### PR TITLE
feat: onboard kubevirt-tekton-tasks

### PR DESCRIPTION
**Pull request by ksimon1**
feat: onboard kubevirt-tekton-tasks
Kubevirt-tekton-tasks are set of tasks, which works with KubeVirt https://kubevirt.io/ and virtual machines and other objects.

### CODE REVIEW

**Comment by ksimon1** [comment]
@savitaashture can you please review this PR?

File: externals.yaml [hunk 1]
```diff
@@ -16,4 +16,7 @@ repositories:
   - tasks
   - pipelines
   ignore-versions: [ "v0.1.0" ]
-
+- url: https://github.com/openshift-cnv/openshift-virtualization-pipelines-tasks
```
Comment (savitaashture) [inline externals.yaml:19]: @ksimon1 I opened https://github.com/openshift-cnv/openshift-virtualization-pipelines-tasks and i don't see tasks there or  am i missing something there :thinking:

Comment (ksimon1) [reply externals.yaml:19]: Tasks are part of release https://github.com/openshift-cnv/openshift-virtualization-pipelines-tasks/releases/tag/4.15.0 - see catalog.yaml, resources.tar.gz.
We are using this approach: https://github.com/openshift-pipelines/tektoncd-catalog/blob/main/docs/workflow-provide-your-tekton-resources.md

Comment (savitaashture) [reply externals.yaml:19]: Thanks for the clarification

**Review by savitaashture — COMMENTED** [review]
/lgtm

**Comment by ksimon1** [comment]
@otaviof can you please review this PR and merge it?

**Review by vdemeester — APPROVED** [review]

**Pull request merged**